Jenkinsfile

```diff
@@ -67,15 +67,39 @@ pipeline {
                                       make rpmlint''',
                            returnStatus: true
                     }
-                }
+                } // stage('RPM Lint')
+                stage('Check Packaging') {
```

Comment:
I have been putting the packaging check inside the Lint block if for no other reason than to parallelize and get done more quickly.

Reply:
I missed adding a parallel stage to that.
It does not look like it buys us much time though.
10 seconds for lint + 5 seconds for Check packaging.